frontend/js/components/bottomPlayerInfo.js

```diff
@@ -0,0 +1,54 @@
+var React = require('react');
+
+var ArimaaStore = require('../stores/ArimaaStore.js');
+var ArimaaActions = require('../actions/ArimaaActions.js');
+var ArimaaConstants = require('../constants/ArimaaConstants.js');
+
+var Utils = require('../utils/Utils.js');
+
+var GameClock = React.createClass({
+  getInitialState: function() {
+    return {
+      clock: ArimaaStore.getBottomClockRemaining(false),
+      playerInfo: ArimaaStore.getBottomUserInfo()
+    };
+  },
+
+  _onChange: function() {
+    this.setState({
+      clock: ArimaaStore.getBottomClockRemaining(false),
+      playerInfo: ArimaaStore.getBottomUserInfo()
+    });
+  },
+
+  componentDidMount: function() {
+    ArimaaStore.addChangeListener(this._onChange);
+    this.interval = setInterval(this._onChange,150);
+  },
+
+  componentWillUnmount: function() {
+    ArimaaStore.removeChangeListener(this._onChange);
+    clearInterval(this.interval);
+  },
+
+  render: function() {
+    //console.log(ArimaaStore.getGameState());
+    var clock = this.state.clock;
+    var formatted = "-:--";
+    if(clock !== null)
+      formatted = Utils.timeSpanToString(Math.max(0,clock));
+
+    var userInfoString = Utils.userDisplayStr(this.state.playerInfo);
+
+    return (
+      <div className={"bottomPlayerInfo"}>
```

Comment:
I assume you were intending to add css for this at some point? What do you think of just sticking in something like uiPanel here, with appropriate extra padding/margin if needed to make it look nice in context?